kitsune/kbforums/events.py

```diff
@@ -103,9 +108,11 @@ def __init__(self, post):
 
     def send_emails(self, exclude=None):
         """Notify watches of the document and of the locale."""
-        return EventUnion(
-            self, NewThreadEvent(self.post), NewThreadInLocaleEvent(self.post)
-        ).send_emails(exclude=exclude)
+        return EventUnion(self, NewThreadInLocaleEvent(self.post)).send_emails(exclude=exclude)
```

Comment:
When working on this, I noticed that the second argument to the `EventUnion`  constructor -- `NewThreadEvent(self.post)` -- was a duplicate of the first -- `self` -- so I removed it.